packages/next/src/server/image-optimizer.ts

```diff
@@ -794,16 +794,3 @@ export async function getImageSize(
   const { width, height } = imageSizeOf(buffer)
   return { width, height }
 }
-
-export class Deferred<T> {
```

Comment:
Not imported 🤷🏻

Reply:
I think this was leftover from something long ago, so it can be removed 😅